tools/pipelines/templates/include-publish-npm-package-steps.yml

```diff
@@ -108,23 +108,23 @@ steps:
                   exit 1
                 fi
             else
-              feedId=$(grep -oP '_packaging/\K[^/]+(?=/npm/)' publish_log)
+              feedName=$(grep -oP '_packaging/\K[^/]+(?=/npm/)' publish_log)
 
-              if [[ $release == "release" && $feedId == "build" ]]; then
+              if [[ "$(release)" == "release" && $feedName == "build" ]]; then
                 packageVersion=$(grep -oP 'npm notice version:\s+\K[^\s]+' publish_log)
 
                 if [ "${{ parameters.artifactPath }}" == "scoped" ]; then
-                  scopes=("fluidframework" "fluid-tools" "fluid-internal" "fluid-experimental")
+                  scopes=("fluidframework" "fluid-tools" "fluid-internal" "fluid-experimental" "fluid-private" "fluid-example")
```

Comment:
Secondary problem: we've changed up our namespaces some. fluid-example is necessary for table-document (unfortunately), and fluid-private got added.